src/views/CampusSafety/views/LostAndFound/views/MissingItemEdit/index.tsx

```diff
@@ -119,6 +124,23 @@ const MissingItemFormEdit = () => {
     setShowConfirm(true);
   };
 
+  const handlePickup = async () => {
+    const requestData = {
+      ...formData,
+      submitterUsername: user.AD_Username,
+      forGuest: false,
```

Comment:
Same here, shouldn't change for Guest, unless there's a specific reason to in this context.

Reply:
Same reason as above, when i pass in the formData, it wants to have the fields for that, unless i was doing it wrongly, but it didnt let me have those out

Reply:
Resolved this, pretty small change.